mods/common/hotkeys/chat.yaml

```diff
@@ -0,0 +1,7 @@
+OpenChat: Return
+	Description: Open Chat
+	Types: Chat, Player, Spectator
+
+OpenAllChat: Return Shift
+	Description: Open All Chat
```

Comment:
TBH I would struggle to guess what "All Chat" meant if it weren't for the context of this PR. Can we please come up with new clearer names for these hotkeys?

Reply:
If "All" isn't clear we could name it "General Chat"?

Reply:
If this is the general chat, shouldn't the other one have the qualifier?

Reply:
> If this is the general chat, shouldn't it just be "Chat" and the other one have the qualifier?

Yes, "Open Chat" and "Open Team Chat" seem like a far better choice. I didn't think this through when I suggested the names to @abmyii :+1:

Reply:
I remember why I proposed this naming - the "Open Team Chat" key actually opens the general chat if the player is not on a team. It serves both functions and that's why I gave the qualifier to the other one :thinking: So maybe "Open Chat" and "Open General Chat" is better after all?

Reply:
Imo "Open Team Chat" opening the general chat if you're not on a team is ok behavioral-wise. (What else would a player expect the key to do when not on a team?)

Reply:
How about `Open Primary Chat` and `Open (General) Chat`?

Reply:
> 
> I agree. How do other games that have separate bindings for opening team/all chat handle this?

I did some research and the most wide-spread configuration seems to be <kbd>Enter</kbd> for team chat, <kbd>Shift</kbd> + <kbd>Enter</kbd> for general chat and <kbd>Tab</kbd> to toggle. Second and third gen CNC games have it as <kbd>Backspace</kbd> for team and <kbd>Enter</kbd> for all. There are some funkier setups like [the prefix system in AoE II](https://gaming.stackexchange.com/a/343641). However I've not been able to see how the "team chat" key behaves when not on a team because it seems I need a real multi-player game to test that. 

After all I think the behavior we've set up here is good and we just need to deal with the labeling problem. How about we make it all explicit: "Open Team Chat" and "Open General Chat"?

Reply:
> How about we make it all explicit: "Open Team Chat" and "Open General Chat"?

This sounds good to me 👍

Reply:
> After all I think the behavior we've set up here is good and we just need to deal with the labeling problem. How about we make it all explicit: "Open Team Chat" and "Open General Chat"?

So that means that the Open Team Chat hotkey (Enter) will only work in team games and will not default to `All` chat?

Reply:
> 
> So that means that the Open Team Chat hotkey (Enter) will only work in team games and will not default to `All` chat?

No, functionality remains the same (team defaults to all if not on a team) - we're only changing the labels and hotkey names.